types/types.go

```diff
@@ -676,6 +677,8 @@ type SystemContext struct {
 	CompressionFormat *compression.Algorithm
 	// CompressionLevel specifies what compression level is used
 	CompressionLevel *int
+	// ProxyURL specifies proxy configuration schema (like socks5://username:password@ip:port)
+	ProxyURL *url.URL
```

Comment:
- Please put this into the “`docker.Transport` overrides” section.
- … and the name should be specific to that transport, maybe `DockerProxyURL`.

Reply:
agree